lib/obfuscate_id.rb

```diff
@@ -22,9 +23,9 @@ def find(*args)
       options = args.slice!(0) || {}
       if has_obfuscated_id? && !options[:no_obfuscated_id]
         if scope.is_a?(Array)
-          scope.map! {|a| deobfuscate_id(a).to_i}
```

Comment:
This is interesting. The find method in the ObfuscateID that my RubyMine is displaying has a totally different implementation for this `find` method. 

`deobfuscate_id` is neither used nor defined in my copy.  

It's showing as 0.0.3